src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -49,3 +34,28 @@
     (utils/get-account-by-address accounts account-address)
     :balance
     (utils/get-balance-by-address balances account-address))))
+
+(defn- calc-token-value
+  [{:keys [symbol market-values-per-currency] :as item}]
+  (let [fiat-value                         (utils/sum-token-chains item)
+        market-values                      (:usd market-values-per-currency)
+        {:keys [price change-pct-24-hour]} market-values
+        fiat-change                        (* fiat-value
+                                              (/ change-pct-24-hour (+ 100 change-pct-24-hour)))]
+    {:token               (keyword (string/lower-case symbol))
+     :state               :default
+     :status              (cond
+                            (pos? change-pct-24-hour) :positive
+                            (neg? change-pct-24-hour) :negative
+                            :else                     :empty)
+     :customization-color :blue
+     :values              {:crypto-value      (.toFixed (* fiat-value price) 2)
+                           :fiat-value        (utils/prettify-balance fiat-value)
+                           :percentage-change (.toFixed change-pct-24-hour 2)
+                           :fiat-change       (utils/prettify-balance fiat-change)}}))
+
+(re-frame/reg-sub
+ :wallet/token-values
```

Comment:
maybe we should be more explicit with the name here? 🤔 
the reason being is that we have token values for all accounts and token value for an account.
so perhaps  `:wallet/token-values-for-account` or something to that effect 🤔